src/pages/storage/forms/StorageVolumeForm.tsx

```diff
@@ -79,6 +83,10 @@ export const volumeFormToPayload = (
       [getVolumeKey("zfs_remove_snapshots")]: values.zfs_remove_snapshots,
       [getVolumeKey("zfs_use_refquota")]: values.zfs_use_refquota,
       [getVolumeKey("zfs_reserve_space")]: values.zfs_reserve_space,
+      ...getUnhandledKeyValues(
+        (volume?.config as LxdConfigPair) || {},
+        getVolumeConfigKeys(),
+      ),
```

Comment:
```suggestion
      ...getUnhandledKeyValues(volume?.config || {}, getVolumeConfigKeys()),
```
better add the `LxdConfigPair` to the `LxdStorageVolume` type and avoid the cast here:
```

  config: {
    "block.filesystem"?: string;
    "block.mount_options"?: string;
    "block.type"?: string;
    "volatile.rootfs.size"?: number;
    "security.shifted"?: string;
    "security.unmapped"?: string;
    "snapshots.expiry"?: string;
    "snapshots.pattern"?: string;
    "snapshots.schedule"?: string;
    "zfs.blocksize"?: string;
    "zfs.block_mode"?: string;
    "zfs.delegate"?: string;
    "zfs.remove_snapshots"?: string;
    "zfs.use_refquota"?: string;
    "zfs.reserve_space"?: string;
    size?: string;
  } & LxdConfigPair;
```